TwitterClone/twitter.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Hello, world!</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <meta name="description" content="" />
+  <script src="https://kit.fontawesome.com/6f98f767cc.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="twitter.css">
+</head>
+<body>
+<div class="container">
+      
+  <div class="left-whitespace">
+    <p></p>
+  </div>
+
+  <div class="sideNav">
+    <div class="logo">
+      <a href="#"><i class="fa-solid fa-cat fa-2xl"></i></a>     
+    </div>
+
+    <div class="navagation">
+      <i class="fa-solid fa-house"></i>
+      <a href="#"><p>Home</p></a>
+      
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-hashtag"></i>
+      <a href="#"><p>Explore</p></a>
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-bell"></i>
+      <a href="#"><p>Notifactions</p></a>
+      
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-envelope"></i>
+      <a href="#"><p>Message</p></a>
+      
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-bookmark"></i>
+      <a href="#"><p>Bookmark</p></a>
+      
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-user"></i>
+      <a href="#"><p>Profile</p></a>
+      
+    </div>
+    <div class="navagation">
+      <i class="fa-solid fa-ellipsis"></i>
+      <a href="#"><p>More</p></a>
+      
+    </div>
+
+    <div class="tweet">
```

Comment:
I noticed that when hovering over it from below it enters a strange loop where he tries to scale down and up! But I don't  know why or how to help with that

Reply:
good spot, don't know why that is will have to check it